lombok.config

```diff
@@ -0,0 +1,2 @@
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
```

Comment:
Config file is added to stop sonar complaining the `@data` test coverage 
https://medium.com/@mladen.bolic/lombok-data-improve-your-code-coverage-a74fb624a72b